engine/src/test/java/io/camunda/zeebe/engine/processing/job/JobThrowErrorTest.java

```diff
@@ -407,7 +407,33 @@ public void shouldTruncateErrorMessage() {
     // then
     Assertions.assertThat(failedRecord).hasRecordType(RecordType.EVENT).hasIntent(ERROR_THROWN);
 
-    final String expectedJobMessage = "*".repeat(MAX_MESSAGE_SIZE).concat("...");
+    final String expectedJobMessage = "*".repeat(DEFAULT_MAX_ERROR_MESSAGE_SIZE).concat("...");
+    assertThat(failedRecord.getValue().getErrorMessage()).isEqualTo(expectedJobMessage);
+
+    final String expectedIncidentMessage =
+        "Expected to throw an error event with the code '' with message '"
+            + expectedJobMessage
+            + "', but it was not caught. No error events are available in the scope.";
+    assertThat(incident.getValue().getErrorMessage()).isEqualTo(expectedIncidentMessage);
+  }
+
+  @Test
+  public void shouldNotTruncateErrorMessage() {
+    // given
+    final var job = ENGINE.createJob(jobType, PROCESS_ID);
+    final String errorMessage = "*".repeat(DEFAULT_MAX_ERROR_MESSAGE_SIZE);
+
+    // when
+    final Record<JobRecordValue> failedRecord =
+        ENGINE.job().withKey(job.getKey()).withErrorMessage(errorMessage).throwError();
+
+    final Record<IncidentRecordValue> incident =
```

Comment:
🙃 Very nitpicky, don't feel pressured to change this if you don't want to, but we could use `var` to make it fit on a single line.

Reply:
Yes, make sense. Thank you